message ResetInfoContainerReading {
    required fixed32 most_recent_reset_cause = 1 [default = 0x0000BBBB];
    required fixed32 most_recent_reset_message = 2 [default = 0];
    required fixed32 num_resets = 3 [default = 0];
    required bytes num_resets_of_cause = 4 [default = ''];
    required bytes num_resets_with_message = 5 [default = ''];
    required fixed32 first_wakeup = 6 [default = 0];
    required bool expected_reset = 7 [default = false];
    required fixed64 timestamp_ms = 8 [default = 0];
}